MultiTarget/WinUI.Extra.props

```diff
@@ -26,7 +26,7 @@
     
     <RuntimeIdentifiers Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) >= 8">win-x86;win-x64;win-arm64</RuntimeIdentifiers> 
     <RuntimeIdentifiers Condition="8 > $([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)'))">win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
-    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+    <WindowsSdkPackageVersion>10.0.22621.37-preview</WindowsSdkPackageVersion>
```

Comment:
We should update to .38 for .NET 6 and .41 for .NET 8 in a follow up

Reply:
Resolved in https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/208/commits/efa78ef1090d1036a8812293292036c885c82218

Reply:
Docs will be in the upcoming CsWinRT release notes, we haven't published them just yet.

Reply:
See https://github.com/microsoft/CsWinRT/releases/tag/2.1.1.240807.2